rest_framework_simplejwt/serializers.py

```diff
@@ -141,4 +146,15 @@ class TokenVerifySerializer(serializers.Serializer):
     def validate(self, attrs):
         UntypedToken(attrs['token'])
 
+        if 'rest_framework_simplejwt.token_blacklist' in settings.INSTALLED_APPS:
+            try:
+                token = RefreshToken(attrs['token'])
+            # not a refresh token
+            except TokenError:
+                return {}
+
+            jti = token.get(api_settings.JTI_CLAIM)
+            if BlacklistedToken.objects.filter(token__jti=jti).exists():
+                raise ValidationError(_('Token is blacklisted'))
```

Comment:
From a security standpoint, you don't want an attacker to know that the stolen token is a refresh token (assuming the token was communicated over an unsafe connection of http).